Optimobile/AppDelegate.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  AppDelegate.swift
+//  Optimobile
+//
+//  Created by Barak Ben Hur on 04/04/2022.
+//
+
+import UIKit
+import KumulosSDK
+
+@main
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
```

Comment:
Im not sure im following yet - this file looks like the AppDelegate for an app and has storyboards etc? Should this all be deleted or is it a test app shipping with the SDK? From the diff it seems to be new.

Reply:
I think these files were used during prototyping, have removed them form the PR.